plugins/catalog-backend/src/service/router.ts

```diff
@@ -88,7 +88,8 @@ export async function createRouter(
   if (higherOrderOperation) {
     router.post('/locations', async (req, res) => {
       const input = await validateRequestBody(req, locationSpecSchema);
-      const output = await higherOrderOperation.addLocation(input);
+      const dryRun = req.query.dryRun === 'true';
```

Comment:
could perhaps use the `yn` lib here to support more truthy syntaxes, but not really necessary :P

Reply:
First I thought: another dependency? but it's already there so: yes yeah!